sopel_remind/plugin.py

```diff
@@ -117,30 +119,37 @@ def remind_in(bot: SopelWrapper, trigger: Trigger):
 
 
 @plugin.command('at')
-@plugin.example('.at 22:15 Do something at 10:15 p.m.', user_help=True)
+@plugin.example('.at 22:15:19 Do something at 10:15 p.m.', user_help=True)
 @plugin.example('.at 10:00 Do something at 10 a.m.', user_help=True)
+@plugin.example('.at 2023-06-27 Python 3.7 EOL.', user_help=True)
+@plugin.example('.at 2023-06-27 12:00:00 Python 3.7 EOL.', user_help=True)
+@plugin.example('.at 12:00:00 2023-06-27 Python 3.7 EOL.', user_help=True)
 def remind_at(bot: SopelWrapper, trigger: Trigger):
-    """Set a reminder for later using hh:mm:ss exact time (timezone aware).
+    """Set a reminder for later using an exact (date) time (timezone aware).
 
     Both hh:mm and hh:mm:ss work. If setting a reminder at a past hour of the
-    day, this will use the same hour the next day. The time uses the same
-    timezone as the user, the channel, or UTC if none is available.
+    day, this will use the same hour the next day. You can set the date using
+    YYYY-MM-DD, either alone, or with a time (before or after).
```

Comment:
This is like my suggestion in the readme. Clearly documenting what the different behaviors are is the goal, but I'm not 100% sure I like this phrasing…

```suggestion
    day, this will use the same hour the next day.

    You can set the date using YYYY-MM-DD, either alone, or with a time
    (before or after). A date-only reminder will use the current time on that
    future date.
```